pageserver/src/tenant.rs

```diff
@@ -1320,7 +1286,7 @@ impl Tenant {
     // This makes the various functions which anyhow::ensure! for Active state work in tests.
     // Our current tests don't need the background loops.
     #[cfg(test)]
```

Comment:
Not this PR's fault, but the comment above could use some love. Aside from the typos, it says "current tests don't need the background loops". I think it refers to the GC and compaction loops, but it's a little confusing because we do launch the layer flush loop below.

Reply:
I'll put that on my list for timeline initialization refactoring